client/src/assets/scss/main.scss

```diff
@@ -11,38 +11,100 @@
 :root {
   -moz-tab-size: 4;
   tab-size: 4;
-}
-
-.dog.cell img {
-  min-height: 268px;
-  object-fit: cover;
-}
-
-h1 {
-  margin: 40px 0.9375rem 20px;
+  --yellow: #ff9505;
+  --orange: #ec4e20;
 }
 
 body,
-div,
-ul,
-p,
+html,
 h1,
 h2,
 h3,
 h4,
 h5,
-h6 {
+h6,
+p,
+div,
+span {
   font-family: "Outfit", "Avenir Next", Avenir, "Helvetica Neue", Helvetica, sans-serif;
 }
 
-h1,
-h2,
-h3,
-h4 {
-  font-weight: 600;
+.dog.cell img {
```

Comment:
Rather than everything thrown into one CSS file, we should start to separate concerns based on the styling we're appling. This main.scss is great for global styles like background color, font colors, etc. But more specific page styling should go in it's own file. We can create stylesheets for separate components or pages (depending how you want to organize things) that way it is easier to find styles to adjust and change later on, instead of all styles meshed in one file. When creating separate SCSS files, you can import them into this `main` file, because this file is already imported into the app